### ISSUE TITLE
Installer fails to save /etc/nipap/nipap.conf, locks itself out of database

### ISSUE THREAD

**Issue by pklausner** [SpriteLink/NIPAP#853]
Ubuntu 1404# apt-get install nipapd nipap-www
* 1st complains about not knowing about postgres, but goes on
* seemingly configures daemon and db backend
* ends with: The specified configuration file ('/etc/nipap/nipap.conf') does not exist
* which is correct: only  nipap.conf.dist exists :(

Retry# dpkg-reconfigure nipapd
* fails with backend    ERROR    pgsql: FATAL:  password authentication failed for user "nipap"   
Backend unable to connect to database
* probably  the random password from the installation is gone with the wind!?

Short of re-installing PostgreSQL, how can I flush the whole DB w/o knowing the password?

**Comment by pklausner** [comment]
For the record: I could recover from this error using:
* pg_dropcluster 9.3 main
* pg_createcluster 9.3 main
* pg_ctlcluster 9.3 main start
* dpkg-reconfigure nipapd

**Comment by plajjan** [comment]
Ok, sorry to hear about the bad experience :/
It comes to prove the process certainly isn't flawless.

I would like to have more debug information if you have.

As for db restoration, it's probably easier to do something like:
```
su - postgres
psql 
ALTER USER nipap WITH PASSWORD 's3cr3t';
```

I'll get back on this one once I've collected my thoughts (and when it's not friday ;).

**Comment by plajjan** [comment]
I think this is addressed by #947. I noticed a flaw in the postinst script where it assumes nipap.conf isn't written when the database is empty. In #947 I recover the old password from nipap.conf if it exists so it shouldn't be able to lock itself out the same way. I will close this issue as  #947 or something very similar will be merged :)
